python/cuspatial/cuspatial/core/gis.py

```diff
@@ -21,53 +21,61 @@ def directed_hausdorff_distance(xs, ys, points_per_space):
     """Compute the directed Hausdorff distances between all pairs of
     spaces.
 
-    params
-    xs: x-coordinates
-    ys: y-coordinates
-    points_per_space: number of points in each space
-
     Parameters
     ----------
-    {params}
+    xs
+        column of x-coordinates
+    ys
+        column of y-coordinates
+    points_per_space
+        number of points in each space
 
-    Example
+    Returns
     -------
+    result : cudf.DataFrame
+        The pairwise directed distance matrix with one row and one
+        column per input space; the value at row i, column j represents the
+        hausdorff distance from space i to space j.
+
+    Examples
+    --------
     The directed Hausdorff distance from one space to another is the greatest
     of all the distances between any point in the first space to the closest
     point in the second.
 
-    [Wikipedia](https://en.wikipedia.org/wiki/Hausdorff_distance)
+    `Wikipedia <https://en.wikipedia.org/wiki/Hausdorff_distance>`_
+
+    Consider a pair of lines on a grid::
+
+             :
+             x
+        -----xyy---
+             :
+             :
```

Comment:
What does this have to do with Hausdorff?

Reply:
I don't know, I assumed you wrote this diagram? I just corrected the formatting.

Reply:
I don’t believe I wrote these docs. Doesn't seem to be relevant to the rest of the example.